manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/persistence/models/Bridge.java

```diff
@@ -0,0 +1,131 @@
+package com.redhat.service.smartevents.manager.v2.persistence.models;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import com.redhat.service.smartevents.manager.core.models.ManagedResource;
+
+@Entity(name = "Bridge_V2")
+@Table(name = "BRIDGE_V2", uniqueConstraints = { @UniqueConstraint(columnNames = { "name", "customer_id" }) })
+public class Bridge extends ManagedResource {
```

Comment:
I would expect a `Operation` field here to easily reach `Condition`

Reply:
We explicitly decided against that and instead will have something like an `OperationService` to handle interaction with operations and conditions. Our reason being two fold: (1) We did not want both a `bridge_id` and `processor_id` FK on the `Operation` table as a single `managed_resource_id` and index is all that is technically needed. This means we cannot have Hibernate handle the relationship. (2) `Operation` is a standalone concept that is applied to a `ManagedResource` by `Work`. It is not a property of the `ManagedResource`.

Reply:
@danielezonca @manstis I've implemented an alternative embedding the Operation in the Bridge/Processor. Check it out https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1354

Reply:
@r00ta 
Thanks for the PR, I personally prefer #1354 so that we avoid at least the additional join. We still have the "fake" join between `Condition` and the managed resource but I don't have any suggestion in particular. If this is going to be a problem we can always have multiple foreign keys so I'm personally in favour to proceed with #1354

Reply:
agreed